README.md

```diff
@@ -18,210 +18,294 @@ Darkflame Universe is licensed under AGPLv3, please read [LICENSE](LICENSE). Som
-```bash
-# Create the build directory, preserving it if it already exists
-mkdir -p build
-cd build
+sudo apt install libssl-dev
 
-# Run CMake to generate build files
-cmake .. -DOPENSSL_ROOT_DIR=/path/to/openssl
+sudo apt install openssl
 
-# Get cmake to build the project. If make files are being used then using make and appending `-j` and the amount of cores to utilize may be preferable, for example `make -j8`
-cmake --build . --config Release
+sudo apt install mariadb-server
+
+# On some systems, this package does not exist.
+sudo apt install zlib
 ```
 
-### Windows builds (native)
-Ensure that you have either the [MSVC](https://visualstudio.microsoft.com/vs/) or the [Clang](https://github.com/llvm/llvm-project/releases/) (recommended) compiler installed. You will also need to install [CMake](https://cmake.org/download/). Currently on native Windows the server will only work in Release mode.
+#### Windows build packages
+Ensure that you have either the [MSVC](https://visualstudio.microsoft.com/vs/) (recommended) or the [Clang](https://github.com/llvm/llvm-project/releases/) compiler installed.
 
-#### Build the repository
-```batch
-:: Create the build directory
-mkdir build
-cd build
+#### MacOS build packages
+Ensure you have [brew](https://brew.sh) installed.
+You will need to install the following packages
+```bash
+brew install cmake
 
-:: Run CMake to generate make files
-cmake ..
+brew install zlib
 
-:: Run CMake with build flag to build
-cmake --build . --config Release
-```
-#### Windows for ARM has not been tested but should build by doing the following
-```batch
-:: Create the build directory
-mkdir build
-cd build
+brew install openssl
 
-:: Run CMake to generate make files
-cmake .. -DMARIADB_BUILD_SOURCE=ON
+brew install gcc
 
-:: Run CMake with build flag to build
-cmake --build . --config Release
+brew install mariadb
 ```
 
-### Windows builds (WSL)
-This section will go through how to install [WSL](https://docs.microsoft.com/en-us/windows/wsl/install) and building in a Linux environment under Windows. WSL requires Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11.
-
-#### Open the Command Prompt application with Administrator permissions and run the following:
-```bash
-# Installing Windows Subsystem for Linux
-wsl --install
-```
+### Database
+Darkflame Universe uses [MariaDB](https://mariadb.org/) for game save data.
 
-#### Open the Ubuntu application and run the following:
+#### Linux MariaDB
+MariaDB is installed in the [Linux build packages](#linux-build-packages) step.  To start the MariaDB server, you will need to run the following command
 ```bash
-# Make sure the install is up to date
-apt update && apt upgrade
-
-# Make sure the gcc, cmake, and build-essentials are installed
-sudo apt install gcc
-sudo apt install cmake
-sudo apt install build-essential
+sudo /etc/init.d/mysql start
 ```
+<font size="4">**You will need to run this command every time you restart your environment**</font>
```

Comment:
Why do this vs `systemctl start mysql`?

Reply:
> Why do this vs systemctl start mysql?

WSL doesn't support systemd, or at least it didn't last time I checked

Reply:
Looks like it's available on newer WSL versions: https://devblogs.microsoft.com/commandline/systemd-support-is-now-available-in-wsl/. TBH neither systemd nor `/etc/init.d/mysql` are portable. It might be better to suggest `mysqld` or `mysqld_safe`, or even link to https://mariadb.com/kb/en/starting-and-stopping-mariadb-automatically/